pyproject.toml

```diff
@@ -36,3 +36,9 @@ urls = { repository = "https://github.com/charliermarsh/ruff" }
 [tool.maturin]
 bindings = "bin"
 strip = true
+
+[tool.setuptools]
+license-files = [
+  "LICENSE",
+  "licenses/*",
+]
```

Comment:
@messense - Do you know if there's any way for Maturin to include these if I'm willing to manually enumerate them? (Even if they're not identified as "licenses" in any special way, but rather, just included in the distributed wheel in some form?)

Reply:
You can use the `tool.maturin.include` option, see https://maturin.rs/metadata.html#add-maturin-build-options

Reply:
Thank you :)

I'm finding that I have to do `include = ["../licenses/*"]` given our structure. Which gives me this slightly strange wheel:

![Screen Shot 2023-01-25 at 1 24 49 PM](https://user-images.githubusercontent.com/1309177/214650260-bad86fce-4f86-4ffc-8743-d1b406dd61c6.png)

Would it be too weird of me to symlink `licenses` into `python`, to get this?
 
![Screen Shot 2023-01-25 at 1 24 24 PM](https://user-images.githubusercontent.com/1309177/214650333-75a7e478-5fb7-4c45-8512-5d6182242ada.png)

Reply:
@messense - Definitely not urgent, I decided to merge the licenses back into a single file (and was able to fix the broken license detector via some licensee hacks, I think).

Reply:
> Would it be too weird of me to symlink `licenses` into `python`, to get this?

I think it's fine for now, in the future `ruff-X.Y.Z.dist-info/license_files` folder is the proper place to put them. `[tool.maturin.include]` doesn't support changing the path in wheel/sdist at the moment.